puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -28,7 +28,7 @@
           description: "Git branch name of Foreman itself, e.g. <pre>develop</pre>"
       - string:
           name: dbs
-          default: 'postgresql mysql sqlite3'
+          default:
```

Comment:
It's useful when running this job directly, it saves users having to enter a value.